### DISCUSSION TITLE
Feature request: Scheduled Reminder for when all CI checks pass

### DISCUSSION THREAD

**Discussion by mbarak** [community/community#39087] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

There is a scheduled reminder to get slack notifications when listed CI checks fail. It would be nice to have the same when all CI checks pass.

Thank you

**Comment by tielur** [comment]
I had this exact same idea this morning. It would be nice to reduce the noise if the PR doesn't pass required checks
